Automattic/jetpack
# Add/pricing info on interstitials for ai and crm

## Proposed changes:

* Show Jetpack AI lowest tier pricing if site is not connected on product interstitial
* Show Jetpack CRM pricing (hardcoded because that info is not available to us via API) on the CRM interstitial
* Send users to CRM pricing page after clicking CTA

### Other information:


## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Using the Jetpack beta plugin or your local environment, checkout this branch
2. Without having AI purchased, go to `/wp-admin/admin.php?page=my-jetpack#/add-jetpack-ai` and make sure you see pricing info
![image](https://github.com/Automattic/jetpack/assets/65001528/5174b7c2-e6ad-4c12-877d-a259109f13b2)
3. Without having Jetpack Complete purchased or the CRM plugin installed, go to `/wp-admin/admin.php?page=my-jetpack#/add-crm` and make sure you can see pricing info
![image](https://github.com/Automattic/jetpack/assets/65001528/9e2102b2-5af1-48ec-a07a-e82956d0c442)
4. Click "Get Jetpack CRM" and make sure the plugin is still installed and you get directed to the CRM pricing page
![image](https://github.com/Automattic/jetpack/assets/65001528/dcca3884-a024-4dda-8a58-0f9eac2bb554)
5. Purchase Jetpack Complete and go back to  `/wp-admin/admin.php?page=my-jetpack#/add-crm` and make sure it says "Install Jetpack CRM" instead of "Get Jetpack CRM". 
![image](https://github.com/Automattic/jetpack/assets/65001528/7e4d5813-2b4b-4202-b5e8-8ebb3acf7c7b)
6. Click on the CTA and make sure you go to the plugin page instead of the pricing page (since CRM is already purchased)
7. Inspect the code carefully to make sure there are no side effects to the changes that I didn't think of
